homeassistant/components/izone/services.yaml

```diff
@@ -0,0 +1,18 @@
+airflow_min:
+  description: Set the airflow minimum percent for a zone
+  fields:
+    entity_id:
+      description: iZone Zone entity
+      example: "climate.bed_1"
+    airflow:
+      description: Airflow percent in 5% increments
+      example: "95"
```

Comment:
```yaml
airflow_min:
  name: Set minimum airflow
  description: Set the airflow minimum percent for a zone
  target:
    entity:
      integration: izone
      domain: climate
  fields:
    airflow:
      name: Percent
      description: Airflow percent in 5% increments
      required: true
      example: "95"
      selector:
        number:
          min: 0
          max: 100
          step: 1
          unit_of_measurement: "%"
          mode: slider
```

The same applies to the service below.

Reply:
For clarification, are you just seeking a documentation PR for this service or something else?

Reply:
Documentation PR added: https://github.com/home-assistant/home-assistant.io/pull/16822

Reply:
> For clarification, are you just seeking a documentation PR for this service or something else?

Documentation yes; but this comment is about this services.yaml file. See example above and the linked developer documentation.

Reply:
Fixed in #47305